### PR TITLE
WebVR documentation

### PR DESCRIPTION
**Pull request by RaananW**
This is an initial version!

### CODE REVIEW

File: content/overviews/Standard/WebVR_Camera.md [hunk 1]
```diff
@@ -0,0 +1,288 @@
...
+
+To understand that think of your head and your body. Without moving your body, your head can move in all directions, and rotate in all directions. The WebVR device is your head. Your body is the regular position and rotationQuaternion we all know and love. If you rotate your body, the head rotates with it. But if you move the head, the body stays in the same position.
+
+This is exactly how you should see the WebVR extra transformation - your head position is set by the VR device (and cannot be interfered with). Your body (or position in the world) is fully controlled by you.
+
+This allows you to use the same code you use for a game based on the FreeCamera with the WebVR camera. the only difference is that the user will have the ability to rotate the camera locally using the VR device and not the mouse.
+
+This also allows the WebVR to be controlled by the same input devices that control the FreeCamera - keyboard, mouse (with rotation exception), XBOX controller and so on. 
+
+### resetting the device's rotation
```
Comment (deltakosh) [inline content/overviews/Standard/WebVR_Camera.md]: capital R? (ok this is all I have to say :D)

**Comment by deltakosh** [comment]
IMPRESSIVE work...Thank you so much!

File: content/overviews/Standard/WebVR_Camera.md (hunk 1 above)
Comment (RaananW) [reply content/overviews/Standard/WebVR_Camera.md]: Oh man... This is bad! Let me fix it :)

Comment (deltakosh) [reply content/overviews/Standard/WebVR_Camera.md]: How dare you submit such a PR!

Comment (RaananW) [reply content/overviews/Standard/WebVR_Camera.md]: I would kick me out if I were you.

Comment (deltakosh) [reply content/overviews/Standard/WebVR_Camera.md]: I'm that close to do it!

Comment (RaananW) [reply content/overviews/Standard/WebVR_Camera.md]: lucky I fixed it 😉

**Comment by deltakosh** [comment]
Thanks my friend! You can merge at will

**Pull request merged**